src/Files.App/UserControls/KeyboardShortcut/KeyboardShortcut.xaml

```diff
@@ -4,46 +4,60 @@
 	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
 	xmlns:local="using:Files.App.UserControls.KeyboardShortcut">
 
-	<local:KeyboardShortcutItemStyleSelector
-		x:Key="KeyboardShortcutItemStyleSelector"
-		AccentItemStyle="{StaticResource AccentKeyboardShortcutItemStyle}"
-		DefaultItemStyle="{StaticResource DefaultKeyboardShortcutItemStyle}"
-		RevealItemStyle="{StaticResource RevealKeyboardShortcutItemStyle}" />
+	<ResourceDictionary.ThemeDictionaries>
+		<ResourceDictionary x:Key="Light">
+			<StaticResource x:Key="OutlinedStyleBackgroundBrush" ResourceKey="ControlFillColorTransparentBrush" />
```

Comment:
Shouldn't this be defined as a SolidColorBrush instead of StaticResource?

Reply:
I'm not sure what the difference is between SolidColorBrush and StaticResource.
Should it be? I saw in this case using StaticResource in Button_themeresources.xaml in MUXC 1.4